detox/src/devices/common/drivers/android/exec/ADB.js

```diff
@@ -135,6 +132,18 @@ class ADB {
     await this.shell(deviceId, `am force-stop ${appId}`);
   }
 
+  async clearAppData(deviceId, packageId) {
+    return await this.shell(deviceId, `pm clear ${packageId}`, { verbosity: 'low', silent: true });
+  }
+
+  async readFile(deviceId, filepath, silent = false) {
+    if (silent) {
```

Comment:
This never seems to be called with `silent=false`. Is this really necessary?
@jonathanmos

Reply:
It's to allow for calling not silently in future. Atm only the optimized flow uses this method

Reply:
It's to allow for calling nonsilently in future. Atm it's called only from the optimized flow